### ISSUE TITLE
[Accessibility]A11y_AzureDataStudio_CreateDatabaseMasterKey_Zoom: Content of the"CreateDatabaseMasterKey" page is disappear when zoom level is 150% and above.

### ISSUE THREAD

**Issue by AccessibilityTestingTeam-TCS** [microsoft/azuredatastudio#5784]
**Bug Description:**

[Check out Keros! ](https://microsoft.sharepoint.com/teams/msenable/Pages/accessibility-tools.aspx)- Identify accessibility bugs before check-in and make bug fixing faster and easier.
GitHubTags: #A11yMAS; #A11yTCS; #MAS1.4.4; #A11y_AzureDataStudio; #MacOs; #DesktopApp
**Environment Details:**
Application name: Azure Data Studio 
App Version: 1.8.0-insider (user setup)
VS Code 1.33.1
Electron: 3.1.8
Node.js: 10.2.0
V8: 6.6.346.32
OS : MacOS Mojave 10.14.5

**Additional Details:**
Tool Name: Zoom
MAS Reference:MAS1.4.4

**Repro Steps:**
1.Launch Visual Studio code desktop application.
2.Navigate to "Enable SQL server2019 preview->select table from the database->Create External Table screen should open->click on next button.
3.Check the content on the page when Zoom is 100%.
4. Zoom page  to 200% to check the content on the page.

**Actual:**
Content of the "CreateDatabaseMasterKey" page is disappear when zoom level is 150% and above.

**Similar issue has been Observed in the below pages:**
**1.CreateConntionDataSrc:** Navigation steps: Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button.
**2.MapDataSrcObjects:** Enable SQL server2019 preview->select table from the database->Create External Table screen should open->Next button->Create Database Master Key screen should open->click on next button->fill all the required details->click on next button.
**3.CSVSelectDestinationDatbse:** Connect to database->navigate to the Data services folder->HDFS Folder-> navigate to CSV files folder->Right click on the files->Select option Create external table from CSV files.
**4. NewJobNotifications** - Go to the SQL server agent tab->On Jobs tab -> Click on New Job button ->Select Notifications tab.
**5. CreateAlert** - Go to the SQL server agent tab->On alerts tab->Click on new alert button 
(Content on the page disappears when zoom level is 125%).
**6. CreateAlertOptions** - Go to the SQL server agent tab->On alerts tab->Click on new alert button -> Select options tab.

**Expected:**
Content of the "CreateDatabaseMasterKey" page should be visible when zoom level is 200%.

**User Impact:**
People with low vision will have difficulties in navigating to the page when Zoom feature is used.

**Recommendation:**
Please refer the following snippet link for fixes:
https://microsoft.sharepoint.com/teams/msenable/mas/Pages/browse-fixes.aspx

**MAS Reference:**
MAS1.4.4
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sou

Attachment for Reference:
[5784_[Accessibility]A11y_AzureDataStudio_CreateDatabaseMasterKey_Zoom.pptx](https://github.com/microsoft/azuredatastudio/files/3241189/5784_.Accessibility.A11y_AzureDataStudio_CreateDatabaseMasterKey_Zoom.pptx)

Does this issue occur when all extensions are disabled?: Yes

**Comment by Charles-Gagnon** [comment]
Looks like the similar issue as https://github.com/microsoft/azuredatastudio/issues/5674
